app.js

```diff
@@ -35,15 +35,25 @@ const { getUserOrCreateUser } = require("./api/helpers/user.js");
 
 const port = process.env.PORT || 3001;
 
+// canonicalize url
+app.use((req, res, next) => {
+  if (
+    process.env.NODE_ENV === "production" &&
+    req.hostname !== "participedia.net"
+  ) {
+    res.redirect("https://participedia.net" + req.originalUrl);
```

Comment:
i think we need a slash at the end of "https://participedia.net", i don't think `req.originalUrl` starts with a slash.

Reply:
just double checked, it does start with a slash, so this looks good!